API/Startup.cs

```diff
@@ -161,17 +162,7 @@ public void ConfigureServices(IServiceCollection services)
                             AuthorizationUrl = new Uri(Config.IdentityServer.IdentityUrl + "/connect/authorize"),
                             Scopes = new Dictionary<string, string>
                             {
-                                { "ProjectWrite", "Project write operations" },
-                                { "ProjectRead", "Project read operations" },
-
-                                { "UserWrite", "User write operations" },
-                                { "UserRead", "User read operations" },
-
-                                { "HighlightWrite", "Highlight write operations" },
-                                { "HighlightRead", "Highlight read operations" },
-                              
-                                { "profile", "Profile information" },
-                                { "openid", "Open id information" }
+                                { "ProjectWrite", "There is a scope needed" },
```

Comment:
What happened here? Why were the others removed?

Reply:
Because Swagger needs a scope, else it will not assign the bearer token to the requests.
but we dont use the token scopes for authorization so the rest are not needed.

Reply:
Okay, can you add a comment to that section of the code for future reference please?

Reply:
i changed it to the resource scope just like with the frontend.